modules/react/avatar/lib/Avatar.tsx

```diff
@@ -139,6 +139,10 @@ const Avatar: AvatarOverload = React.forwardRef(
       }
     };
 
+    React.useEffect(() => {
```

Comment:
This might cause rendering issues in the UI visual regression tests. I'll switch to a `React.useLayoutEffect` to make sure there isn't a frame with an incorrect image. This is technically able to change the `imageLoaded` to `false` if the image is loaded and available _before_ the `onLoad` of the image is called. I'm seeing this only in IE11, but it is a race condition.

```suggestion
    React.useLayoutEffect(() => {
```